docs/manuals/source/manually_added_config_directive_descriptions/dir-director-PidDirectory.rst.inc

```diff
@@ -1,5 +1,7 @@
-This directive is optional and specifies a directory in which the Director may put its process Id file. The process Id file is used to shutdown Bareos and to prevent multiple copies of Bareos from running simultaneously. Standard shell expansion of the Directory is done when the configuration file is read so that values such as $HOME will be properly expanded.
+This directive is optional and specifies a directory in which the Director may put its process Id file. The process Id file is used to shut down Bareos and to prevent multiple copies of Bareos from running simultaneously. Standard shell expansion of the Directory is done when the configuration file is read so that values such as $HOME will be properly expanded.
 
 The PID directory specified must already exist and be readable and writable by the Bareos daemon referencing it.
 
 Typically on Linux systems, you will set this to: :file:`/var/run`. If you are not installing Bareos in the system directories, you can use the Working Directory as defined above.
+
+As of bareos 21, this directive is deprecated. The way to set up a pid file is to do it as an option to the Director binary with the ``-p <file>`` option, where <file> is the path to a pidfile of your choosing. By default, no pidfile is created.
```

Comment:
```suggestion
Since :sinceVersion:`21.0.0` this directive has no effect anymore. The way to set up a pid file is to do it as an option to the Director binary with the ``-p <file>`` option, where <file> is the path to a pidfile of your choosing. By default, no pidfile is created.
```

Reply:
Makes sense, but shouldn't we keep the old description for users of previous versions?